src/widgets/ot_color_picker.eliomi

```diff
@@ -23,9 +23,19 @@
 
 (** This module implements a color picker.  *)
 
-val hsv_to_rgb : int -> float -> float -> (float * float * float)
+val hsv_to_rgb : int -> float -> float -> float * float * float
 (** [hsv_to_rgb h s v] converts HS(V/L) colors to RGB. *)
 
-val make : ?a: [< Html_types.div_attrib > `Class ] Eliom_content.Html.attrib list ->
-           [> `Div ] Eliom_content.Html.D.elt * (int * float * float) Eliom_shared.React.S.t
-(** [make ()] produces a color picker. *)
+val make :
+  ?a:[< Html_types.div_attrib > `Class ] Eliom_content.Html.attrib list ->
+  ?hsv:int * float * float ->
+  ?update:(int * float * float) React.E.t Eliom_client_value.t ->
+  unit ->
+  [> `Div ] Eliom_content.Html.D.elt
+  * (int * float * float) Eliom_shared.React.S.t
+(** [make ()] produces a color picker. 
+    [?a] is an optional list of html class, by default it is the empty list 
```

Comment:
is an optional parameter to add html attributes to main element

(not a list of classes)